src/platform/web/transport/transport.ts

```diff
@@ -252,7 +252,8 @@ export class Transport implements TransportDefinition {
     try {
       // WebSocket()
       // https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket
-      ws = new WebSocket(this.server, "sip");
+      const transportProtocols = [...new Set(this.configuration.transportProtocols)];
+      ws = new WebSocket(this.server, transportProtocols);
```

Comment:
@kkbhanuchandar why not add a getter for transportProtocols, like there is for server and then just do?
`ws = new WebSocket(this.server, this.transportProtocols);`

Reply:
Don't think this is really necessary but it is something that can be done 😊